copy/src/components/QuestionCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {AnswerObject} from '../App'
+import {Wrapper} from './QuestionCard.Styles'
+
+type Props = {
+  question: string;
+  answers: string[];
+  callback: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  userAnswer: AnswerObject | undefined ;
```

Comment:
you can mark prop as not required by adding ? to it's name instead of `| undefined`, eg
`userAnswer?: Answer`

Also, don't add type name to... type name :D I mean, better "Answer" instead of "AnswerObject"
 Things like "object", "data", "item", etc are not good names

Aaaalso, Props should be an interface (although not technically wrong to have a type)